### DISCUSSION TITLE
How to create overlay in Fo-Dicom, need code example to create and show overlay layer using fo-dicom

### DISCUSSION THREAD

**Discussion by BKotarki** [fo-dicom/fo-dicom#1906] [General]
How to create overlay in Fo-Dicom, need code example to create and show overlay layer using fo-dicom

**Comment by jaime-olivares** [comment]
This issue seems to be related: https://github.com/fo-dicom/fo-dicom/issues/770 

You can create a new DICOM file from scratch containing a vectorial or bitmap overlay (GSPS) and relate it to a specific image in the study.
It will depend on your viewer whether it can present that conformant overlay.
